src/contracts/facilitators/aave/stkAaveUpgrade/interfaces/IStkAave.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IStkAave {
```

Comment:
This interface seems already imported in the `safety-module` dependency. Can we fetch it from there?

Reply:
needed to re-write due solidity version is 6 or 7, so is not compatible if the test suite runs on Solidity 8